networkx/algorithms/clique.py

```diff
@@ -578,7 +578,45 @@ def number_of_cliques(G, nodes=None, cliques=None):
 
     Returns a single or list depending on input nodes.
     Optional list of cliques can be input if already computed.
+
+    Parameters
+    ----------
+    G : NetworkX graph
+        An undirected graph.
+
+    nodes: list, optional (default=None)
```

Comment:
Technically `nodes` can be a single node as well: the type listing here and parameter description should be updated to reflect this.